plugins/orchestrator-backend/src/service/SonataFlowService.ts

```diff
@@ -104,15 +103,18 @@ export class SonataFlowService {
         ? `${args.serviceUrl}/${args.definitionId}?businessKey=${args.businessKey}`
         : `${args.serviceUrl}/${args.definitionId}`;
 
-      const result = await fetch(urlToFetch, {
+      const response = await fetch(urlToFetch, {
         method: 'POST',
         body: JSON.stringify(args.inputData),
         headers: { 'content-type': 'application/json' },
       });
 
-      const json = await result.json();
-      this.logger.debug(`Execute workflow result: ${JSON.stringify(json)}`);
-      return json;
+      if (response.ok) {
+        const json = await response.json();
+        this.logger.debug(`Execute workflow result: ${JSON.stringify(json)}`);
+        return json;
+      }
+      throw new Error(this.getFetchErrorMessage(urlToFetch, 'executeWorkflow'));
```

Comment:
I would have used `POST ` instead of `executeWorkflow` as the REST method is POST

Reply:
I'm ok with that, but not sure if the context of this PR is the right one. I think it's more a refectory activity and it should be "extended" to other functions in this file. WDYT?

Reply:
ok, let's go with that, please create a follow up issue :)

Reply:
I actually think executeWorkflow is a good name for the function. It explains exactly what this function does. The POST is an implementation detail

Reply:
I am also in favor of putting the name of the operation and not the HTTP method. But...it is easy to put both: 'executeWorkflow (POST)'. The more info the better